misk/src/main/kotlin/misk/web/interceptors/RequestBodyLoggingInterceptor.kt

```diff
@@ -90,9 +90,7 @@ internal class RequestResponseCapture @Inject constructor() {
     private val capture = ThreadLocal<RequestResponseBody>()
   }
 
-  fun get(): RequestResponseBody {
-    return capture.get()
-  }
+  fun get(): RequestResponseBody? = capture?.run { get() }
```

Comment:
Something like this:
```suggestion
  fun get(): RequestResponseBody? {
    if (capture.isPresent()) {
      return capture.get()
    }
    return null;
  }
```

Reply:
Ah, no, this isn't quite right either.